scrape_all_prs.py

```diff
@@ -104,8 +107,10 @@ def collect_pr_metadata():
                 pr.updated_at, 
                 "merged" if pr.merged else "closed", 
                 pr.changed_files, 
-                len([comment for comment in pr.get_comments() if comment.user.type != "Bot"]), 
+                sum([file.changes for file in pr.get_files()]),
```

Comment:
Used to get lines of code changed (file.changes is sum of additions+deletions, and sum this for each file in PR)